main.go

```diff
@@ -130,15 +132,48 @@ func (c *runCmd) Run() error {
 	if err != nil {
 		return err
 	}
-	rt := cli.NewRuntime()
-	rt.SkipSleep = c.SkipSleep
+
+	rt := cli.NewRuntime(c.runtimeOptions()...)
+
 	eval := evaluator.NewEvaluator(rt)
-	err = eval.Run(string(b))
-	handlEvyErr(err)
+	evyErr := eval.Run(string(b))
+	if errors.As(evyErr, &parser.Errors{}) {
+		handleEvyErr(evyErr)
+	}
+	if err := c.writeSVG(rt); err != nil {
+		handleEvyErr(err)
+	}
+	handleEvyErr(evyErr) // may exit
```

Comment:
consider writing this as:
```
evyErr := eval.Run(string(b)
if !errors.As(evyErr, &parser.Errors{}) {
    // even if there was an evaluator error, we want to write as much of the SVG that was produced.
    err = c.writeSVG(rt)
}
handleEvyErr(cmp.Or(evyErr, err))
```

Reply:
nice, didn't know about cmp.Or - that's cool. Done.